src/TokenParser/Render.php

```diff
@@ -46,7 +46,7 @@ class Render extends Twig_TokenParser {
    *
    * @see Twig_TokenParser_Include::parse()
    */
-  public function parse(Twig_Token $token) {
+  public function parse(Twig_Token $token): \Twig_Node_Include {
```

Comment:
Shouldn't the expected return value be a `Node\Render`?

Reply:
Since `Node\Render` extends `Twig_Node_Include` the return type is correct. Is it better to declare the actual one than the parent?

Reply:
It's a matter of expectations… and I'd expect our Render TokenParser to parse into a Render Node.

Reply:
Ok, make sense to me.